framework/src/controller/helpers/validator/formats.js

```diff
@@ -0,0 +1,359 @@
+const ip = require('ip');
```

Comment:
Tests for this new file need to be moved from `framework/test/mocha/unit/modules/chain/helpers`. I will create an issue for that.